backend/app.js

```diff
@@ -9,7 +9,7 @@ const morgan = require('morgan');
 const cookieParser = require("cookie-parser");
 
 const customRequestHeaderName = 'x-customrequired-header';
-const dontCheckCustomRequestHeaderApis = ["GET::/api/recurringevents"];
+const dontCheckCustomRequestHeaderApis = ["GET::/api/recurringevents", "GET::/api/healthcheck"];
```

Comment:
Given our recent experience getting `api/recurringevents` to work, should this be `/api/recurringEvents`?  That this worked on your machine and not mine makes me think it's probably fine as is, but I'm just putting it out there.

Reply:
@drubgrubby it's about the internal checking function so we can ignore the case of the letter here.